app/overrides/layouts/base/eol_banner.html.erb.deface

```diff
@@ -21,7 +21,7 @@ end %>
     <% end %>
     <div class="eol-banner-content">
     <%= _("For up to date information about Satellite's lifecycle see ") %><a href="https://access.redhat.com/support/policy/updates/satellite"><%= _("Red Hat Satellite Product Life Cycle") %></a>.
-    <%= _("Kindly plan to upgrade Satellite to the supported version, for upgrade guidance see ") %><a href="<%= ForemanThemeSatellite.documentation_server %>/labsinfo/satelliteupgradehelper "><%= _("Red Hat Satellite Upgrade Helper") %>.
```

Comment:
It's also used in https://github.com/RedHatSatellite/foreman_theme_satellite/blob/8521fb77ddb314501cc425ba95351031058dd999/app/overrides/satellite_name_override.rb#L40. Should we remove that method at all since there are 2 usages only?

Or maybe change the default value to be `https://access.redhat.com` instead of `https://docs.redhat.com`?

Don't take this as a request, but rather a question to learn about this plugin more :)

Reply:
> . Should we remove that method at all since there are 2 usages only?

Behind the VPN there is also a staging URL for docs so you can reconfigure the documentation server to point to it.

> Or maybe change the default value to be `https://access.redhat.com` instead of `https://docs.redhat.com`?

That would revert https://github.com/RedHatSatellite/foreman_theme_satellite/commit/cf39ce7e68b684cf368b7743a01ddb8d224a698d.

Reply:
> It's also used in [foreman_theme_satellite/app/overrides/satellite_name_override.rb](https://github.com/RedHatSatellite/foreman_theme_satellite/blob/8521fb77ddb314501cc425ba95351031058dd999/app/overrides/satellite_name_override.rb#L40)

I think it is fine in there, the docs do really live at docs.rh.com

> Or maybe change the default value to be https://access.redhat.com instead of https://docs.redhat.com?

That would essentially undo  https://github.com/RedHatSatellite/foreman_theme_satellite/pull/59 :/